### DISCUSSION TITLE
POI update (delete or not delete other tags)

### DISCUSSION THREAD

**Discussion by Filip10** [Helium314/SCEE#670] [Q&A] [answered]
I remember, that when I was changing the object, app asked me about "if it is still the same thing" but now it is not asking me and I don't know what I have changed.

**Comment by mnalis** [comment]
There is somewhat complicated logic behind when that is asked:

https://github.com/Helium314/SCEE/blob/a5043084be37f44bbe5bd5529b4d2e52157c1f5a/app/src/main/java/de/westnordost/streetcomplete/overlays/places/PlacesOverlayForm.kt#L271-L298

but basically, in _most cases_ it will get asked _"if it is still the same thing"_ in Places Overlay if:

- you change only the name  (but type of the place remain the same), _or_
- place had no name before, and you changed type of the place

**Comment by Filip10** [comment]
Because I'm changing some POI's, which are included in Name Suggestion Index, and when I change some just to add operator:wikidata and operator, it is also deleting opening_hours and other tags, which is not what I want. I just tested it out with name or place type change and question _"if it is still the same thing"_ was not asked.

**Reply by mnalis** [reply, marked as answer]
Thanks for clarifying, that is indeed unfortunate! Given that the issue is the same in upstream StreetComplete, I've made an enhancement request there to improve that use case, so you may want to subscribe/like there: https://github.com/streetcomplete/StreetComplete/issues/5940

**Reply by Filip10** [reply]
Thank you :)
